### ISSUE TITLE
`schema inspect` does not return views?

### ISSUE THREAD

**Issue by sollie** [ariga/atlas#2157]
It looks like support for views was added in 0.13, but when running schema inspect on MySQL I don't see my views. My user can list views in information_schema.tables and can run `show create table`.

Am I missing something in the documentation, do I need to add a flag to dump views?

**Comment by sollie** [comment]
```
❯ /opt/homebrew/bin/atlas version
atlas version v0.14.2
https://github.com/ariga/atlas/releases/tag/v0.14.2
```

and with `go install` it's simply 

```
❯ ~/go/bin/atlas version
atlas version - development
https://github.com/ariga/atlas/releases/latest
```

**Comment by a8m** [comment]
Using plain `go build` does not contain this functionality. Installing Atlas as explained in the website should solve the issue: https://atlasgo.io/getting-started#installation

**Comment by sollie** [comment]
Apologies.

Since I read that it was released in 0.13, I figured 0.14 from homebrew (not your tap, though) would contain it.

Reinstalled with `brew install ariga/tap/atlas` and it seems to work.
Thanks for all the work you've put into this!
